util/lib.py

```
@@ -342,41 +343,69 @@
 
-def robot_prepare_ontology(o_path, o_out_path, o_metrics_path, base_iris, make_base, robot_prefixes={}, robot_opts="-v"):
-    logging.info(f"Preparing {o_path} for dashboard.")
-    
-    callstring = ['robot', 'merge', '-i', o_path]
-    
+def robot_prepare_ontology(
+    o_path: str,
+    o_out_path: str,
+    o_metrics_path: str,
+    base_iris: List[str],
+    make_base: bool,
```

Ruff

Boolean-typed positional argument in function definition

(FBT001)